libs/logic-apps-shared/src/utils/src/lib/models/run.ts

```diff
@@ -52,17 +52,21 @@ export interface SecureData {
   properties: string[];
 }
 
-export type BoundParameters = Record<string, BoundParameter<any>>;
+export type BoundParameters = Record<string, BoundParameter>;
 
-interface BoundParameter<T> {
+export interface BoundParameter {
   displayName: string;
-  dynamicValue?: T;
+  dynamicValue?: any;
   format?: string;
   language?: string;
-  value: T;
+  value: any;
   visibility?: string;
 }
 
+export type BindFunction = (values: any, parameter: any) => BoundParameter | undefined;
```

Comment:
Why is yhe parameter of type any is this InputParameter or ParameterInfo?

Reply:
Yep, its an inputParameter. I will change it